app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/PrefsFragment.kt

```diff
@@ -0,0 +1,88 @@
+ *
+ * This file is part of Amaze File Manager.
+ *
+ * Amaze File Manager is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager.ui.fragments.preference_fragments
+
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.widget.Toast
+import androidx.preference.Preference
+import com.amaze.filemanager.R
+import com.amaze.filemanager.ui.activities.AboutActivity
+import com.amaze.filemanager.utils.Utils
+
+class PrefsFragment : BasePrefsFragment() {
+    override val title = R.string.setting
+
+    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
+        setPreferencesFromResource(R.xml.preferences, rootKey)
+
+        findPreference<Preference>("appearance")!!.onPreferenceClickListener =
+                Preference.OnPreferenceClickListener {
+                    activity.pushFragment(AppearancePrefsFragment())
+                    true
+                }
+
+        findPreference<Preference>("ui")!!.onPreferenceClickListener =
+                Preference.OnPreferenceClickListener {
+                    activity.pushFragment(UiPrefsFragment())
+                    true
+                }
+
+        findPreference<Preference>("behavior")!!.onPreferenceClickListener =
+                Preference.OnPreferenceClickListener {
+                    activity.pushFragment(BehaviorPrefsFragment())
+                    true
+                }
+
+        findPreference<Preference>("security")!!.onPreferenceClickListener =
+                Preference.OnPreferenceClickListener {
+                    activity.pushFragment(SecurityPrefsFragment())
+                    true
+                }
+
+        findPreference<Preference>("about")!!.onPreferenceClickListener =
+                Preference.OnPreferenceClickListener {
+                    startActivity(Intent(activity, AboutActivity::class.java))
+                    false
+                }
+
+        findPreference<Preference>("feedback")!!
+                .onPreferenceClickListener = Preference.OnPreferenceClickListener {
+            val emailIntent = Utils.buildEmailIntent(null, Utils.EMAIL_SUPPORT)
+
+            val activities = activity.packageManager.queryIntentActivities(
+                    emailIntent, PackageManager.MATCH_DEFAULT_ONLY)
```

Comment:
`spotlessCheck` fails here

```
Step 'ktlint' found problem in 'app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/PrefsFragment.kt':
Error on line: 73, column: 38
Exceeded max line length (100)
```

I tried even making this whole callback into a comment and it didn't work. No idea why this is failing here. Any ideas?

Reply:
> 



> `spotlessCheck` fails here
> 
> ```
> Step 'ktlint' found problem in 'app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/PrefsFragment.kt':
> Error on line: 73, column: 38
> Exceeded max line length (100)
> ```
> 
> I tried even making this whole callback into a comment and it didn't work. No idea why this is failing here. Any ideas?

It's exceeding the max line length - try renaming the variables to something shorter & see if that works.

Reply:
Problem solved, the issue was another line down below that was too large. Weird that the number was completely off.